cli/command/trust/sign.go

```diff
@@ -21,7 +21,7 @@ import (
 
 func newSignCommand(dockerCli command.Cli) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "sign [OPTIONS] IMAGE:TAG",
+		Use:   "sign IMAGE:TAG",
```

Comment:
sign is about to have the --local flag, is it worth removing it just to add it right back?

Reply:
let's add it in the `--local` flag PR explicitly.  That PR could even go against master once we're ready